zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/ParagraphRuntimeInfos.java

```diff
@@ -0,0 +1,29 @@
+package org.apache.zeppelin.notebook;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * 
+ *
+ */
+public class ParagraphRuntimeInfos {
+
+  String propertyName;
+  String label;
+  String group;
```

Comment:
The label is used here 
https://github.com/apache/zeppelin/blob/9612dd08b12b1b76f70d9baab1064d0207f1fd46/zeppelin-web/src/app/notebook/paragraph/paragraph-control.html#L19
I had added group , so that two interpreters having same property name can be rendered in a different manner in UI.